### DISCUSSION TITLE
Test logs not shown in IDE

### DISCUSSION THREAD

**Discussion by StefanH-AT** [MikeSchulze/gdUnit4#426] [Q&A]
According to this discussion on version 3, the output is only shown when running with debugger, but this doesn't work for me. Using rider here where the debugger seems busted in general. Is that related?

https://github.com/MikeSchulze/gdUnit3/discussions/212

I like using unit tests while creating features and often use the output during development before writing assets. this would be really useful in all scenarios

**Comment by MikeSchulze** [comment]
Have you checked your `.runsetting` there is a section to configure the log level.
```
    <LoggerRunSettings>
        <Loggers>
            <Logger friendlyName="console" enabled="True">
                <Configuration>
                    <Verbosity>detailed</Verbosity>
                </Configuration>
            </Logger>
        </Loggers>
    </LoggerRunSettings>
```
